classes/PodsData.php

```diff
@@ -2789,7 +2789,12 @@ public static function query_field( $field, $q, $pod = null, &$params = null ) {
 								$table = $the_field->get_table_info();
 
 								if ( ! empty( $table ) ) {
-									$field_cast = "`{$field_name}`.`" . $table['field_id'] . '`';
+									if ( is_int( $field_value ) ) {
```

Comment:
Should we check numeric or just int?

```suggestion
									if ( is_numeric( $field_value ) ) {
```

Reply:
is_numeric wouldn't be backwards compatible.  The query string I'm using is '2015'